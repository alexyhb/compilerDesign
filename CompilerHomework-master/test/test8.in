prefix$var"$anothervar" "all one string"
";" \; ; "|"\||
"one string$split another string"
