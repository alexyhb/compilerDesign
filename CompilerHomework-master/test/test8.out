Text ->prefix<-
Var ->var<-
Var ->anothervar<-
Blank 1 chars
DblQuoted ->all one string<-
NL
DblQuoted ->;<-
Blank 1 chars
Text ->\;<-
Blank 1 chars
Semi
Blank 1 chars
DblQuoted ->|<-
Text ->\|<-
Pipe
NL
DblQuoted ->one string<-
Var ->split<-
DblQuoted -> another string<-
NL
